.github/workflows/build.yaml

```diff
@@ -0,0 +1,31 @@
+name: Func Build
```

Comment:
Is splitting all the github actions necessary to achieve Go 1.16 and Knative 0.25? If not I'd really try to keep these concerns separate.

Reply:
In fact, if possible, I'd even keep bumping the Knative version seperate from bumping the Golang version.

Reply:
I am sorry it is difficult to read, I will keep things more separate going forward.